types/react-native/v0.64/index.d.ts

```diff
@@ -6930,7 +6930,7 @@ export interface ActionSheetIOSOptions {
     title?: string | undefined;
     options: string[];
     cancelButtonIndex?: number | undefined;
-    destructiveButtonIndex?: number | undefined;
+    destructiveButtonIndex?: number | number[] | undefined;
```

Comment:
We could improve it here, as Maybe types (see the original source code written in Flow) would also accept `null`: https://flow.org/en/docs/types/maybe/
```suggestion
    destructiveButtonIndex?: number | number[] | undefined | null;
```